apis/management.cattle.io/v3/authn_types.go

```diff
@@ -143,6 +144,15 @@ type AuthConfig struct {
 	AllowedPrincipalIDs []string `json:"allowedPrincipalIds,omitempty" norman:"type=array[reference[principal]]"`
 }
 
+type SamlToken struct {
+	types.Namespaced
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Token     string `json:"token"`
```

Comment:
Maybe this can have these tags: ```norman:"writeOnly,noupdate"```
Similar to https://github.com/rancher/types/blob/08e9d40fbb96851066328470b362b4a0d371fc93/apis/management.cattle.io/v3/authn_types.go#L16